apps/backoffice-v2/src/common/components/molecules/GetFullAccessCard/GetFullAccessCard.tsx

```diff
@@ -15,7 +15,7 @@ export const GetFullAccessCard = () => {
   if (
     env.VITE_ENVIRONMENT_NAME === 'production' ||
```

Comment:
We are going to create the demo accounts in production, and not sandbox this time, as this environment is more stable. Can you please remove this condition and make sure it works well on a production environment?

Context: https://teamballerine.slack.com/archives/C06MQM4F17D/p1739886859740529